scripts/builtin/tSNE.dml

```diff
@@ -119,7 +161,7 @@ return(matrix[double] P)
   while (mean(abs(Hdiff)) > tol & itr < 50) {
     P = exp(-D * beta)
     P = P * ZERODIAG
-    sum_Pi = rowSums(P)
+    sum_Pi = rowSums(P) = 1e-12
```

Comment:
this line confuses me, what is the intention?

Reply:
My mistake, this should be `sum_Pi = rowSums(P) + 1e-12`. This is to prevent zero division errors that came up multiple times during the experiments.